docs/pages/admin-guides/migrate-plans.mdx

```diff
@@ -219,6 +228,21 @@ To migrate Teleport agents:
 
 1. Stop Teleport services on each Agent (if applicable).
 
+1. If you are using Teleport repositories to install on Linux
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (Cloud). For self-hosted Teleport Enterprise you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using automatic
+   upgrades the `stable/rolling` channel has all released versions.
+
+
+1. If you are using the Teleport Community Edition, [uninstall Teleport](./management/admin/uninstall-teleport.mdx)
+   and install the enterprise version. You can confirm if the `teleport` binary
+   is enterprise by running `teleport version`. The output will contain the words
+   `Teleport Enterprise`. The enterprise version of the `teleport`

```

Comment:
Should `teleport` here be `tctl`, since we're talking about connecting to a Teleport cluster?